### PR TITLE
clean up footer

### PR DESCRIPTION
**Pull request by ciaranschutte**

### CODE REVIEW

**Review by justincorrigible — APPROVED** [review]

File: src/app/components/Footer.tsx [hunk 1]
```diff
@@ -19,49 +19,130 @@
...
+import rdpcLogo from '/public/assets/rdpc-logo.svg';
 
 const { UI_VERSION, REGION } = getAppConfig();
 const subtitle = `RDPC ${REGION} Clinical Data Submission Portal - ${UI_VERSION}`;
 
-const Logo = () => <div>RDPC</div>;
+const Logo = () => <Image src={rdpcLogo} alt="RDPC logo" />;
 
-export default function GlobalFooter() {
+const links = [
```
Comment (justincorrigible) [inline src/app/components/Footer.tsx:33]: this could go into a separate (local) constants file

File: src/app/components/Footer.tsx [hunk 2]
```diff
@@ -19,49 +19,130 @@
...
+						padding-left: 22px;
+					`}
+				>
+					{/** nav links */}
+					<div
+						css={css`
+							margin-right: 22px;
+						`}
+					>
+						{links.map(({ displayName, href, target }, idx) => (
```
Comment (justincorrigible) [inline src/app/components/Footer.tsx:122]: snark for comedic tone, not for snub:
how is idx better than index? "3 > 5" economy?

File: src/app/components/Footer.tsx [hunk 3]
```diff
@@ -19,49 +19,130 @@
...
+									target={target}
+									href={href}
+									rel={target === '_blank' ? 'noreferrer noopener' : ''}
+									css={css`
+										color: ${theme.colors.primary};
+									`}
+								>
+									{displayName}
+								</Link>
+								{idx !== links.length - 1 && (
```
Comment (justincorrigible) [inline src/app/components/Footer.tsx:134]: having a slash divider be an icon and adding it through array index logic seems like an overengineered alternative to an easy CSS solution

Comment (ciaranschutte) [reply src/app/components/Footer.tsx:134]: I agree

**Pull request merged**